### DISCUSSION TITLE
Support Wintertodt Warmth Meter in Status Bars

### DISCUSSION THREAD

**Discussion by coopermor** [runelite/runelite#18422] [Ideas]
With the changes to Wintertodt HP is not a very useful mechanic. It would be nice if Status Bars could be updated with an option for HP to change to Warmth Meter when inside wintertodt.

**Comment by ctdgunner** [comment]
Was literally coming to request this just now. Please :)

**Comment by RickRokner** [comment]
Any update to this yet?

**Comment by Adam-** [comment]
I've added this for the next update
![image](https://github.com/user-attachments/assets/f5e8eeb1-52e1-496d-9542-fca8beec7149)

**Comment by JessofHearts** [comment]
Is it possible to have the warmth status bar highlight how much would be healed when hovering over a rejuvenation potion like food with the HP bar?

**Reply by ldavid432** [reply]
I have a PR up to add that: https://github.com/runelite/runelite/pull/18705
